geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/utils/CoprocessorBatchScan.scala

```diff
@@ -13,31 +13,35 @@ import java.util.concurrent._
 import org.apache.hadoop.hbase.TableName
 import org.apache.hadoop.hbase.client._
 import org.locationtech.geomesa.hbase.HBaseSystemProperties
+import org.locationtech.geomesa.hbase.data.HBaseQueryPlan
 import org.locationtech.geomesa.hbase.rpc.coprocessor.GeoMesaCoprocessor
 import org.locationtech.geomesa.index.utils.AbstractBatchScan
+import org.locationtech.geomesa.index.utils.ThreadManagement.{LowLevelScanner, ManagedScan, Timeout}
 import org.locationtech.geomesa.utils.collection.CloseableIterator
 import org.locationtech.geomesa.utils.concurrent.CachedThreadPool
-import org.locationtech.geomesa.utils.io.WithClose
+import org.opengis.filter.Filter
 
 private class CoprocessorBatchScan(
     connection: Connection,
     table: TableName,
     ranges: Seq[Scan],
     options: Map[String, String],
-    threads: Int,
     rpcThreads: Int,
     buffer: Int
-  ) extends AbstractBatchScan[Scan, Array[Byte]](ranges, threads, buffer, CoprocessorBatchScan.Sentinel) {
+  ) extends AbstractBatchScan[Scan, Array[Byte]](ranges, rpcThreads * 2, buffer, CoprocessorBatchScan.Sentinel) {
```

Comment:
For this commit / PR, should this be 'threads'?

Reply:
i think the previous behavior was ranges.length, should i revert it? i didn't bother since i added the other pr to update it.